fhir_river_api/src/app.py

```diff
@@ -14,30 +13,47 @@
 
 # Create flask app object
 app = Flask(__name__)
-api = Api(app)
 
 producer = RiverApiProducer(broker=os.getenv("KAFKA_BOOTSTRAP_SERVERS"))
+EXTRACTOR_URL = os.getenv("EXTRACTOR_URL")
+TRANSFORMER_URL = os.getenv("TRANSFORMER_URL")
 
 
-@app.route("/run_sample", methods=["POST"])
+@app.route("/preview", methods=["POST"])
 def trigger_sample_extractor():
     """
     Extract record for the specified resource and primary key value
     """
     body = request.get_json()
     resource_id = body.get("resource_id", None)
     primary_key_values = body.get("primary_key_values", None)
```

Comment:
Make it fail if no `primary_key_values`. 
Otherwise, it will run all the batch

Reply:
indeed. done.